nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayJava8PositiveCases.java

```diff
@@ -59,6 +62,4 @@ static void testNullableParam() {
         NonNullParamFunction n3 = (@Nullable Object x) -> x.toString();
     }
 
```

Comment:
Shouldn't we have some new positive test cases?

Reply:
We have the one in `NullAwayRxSupportPositiveCases`, but just added a few on `NullAwayJava8PositiveCases` as well.